Say Their Names/Source/Controller/Home/HomeController.swift

```diff
@@ -104,7 +104,8 @@ class HomeController: UIViewController, ServiceReferring {
         
         peopleCollectionView.delegate = self
         peopleCollectionView.dataSource = peopleDataSource
-        peopleCollectionView.register(UINib(nibName: peopleIdentifier, bundle: nil), forCellWithReuseIdentifier: PersonCell.personIdentifier)
+        peopleCollectionView.register(PersonCell.self, forCellWithReuseIdentifier: PersonCell.personIdentifier)
```

Comment:
I think it would be nice to simply this
```suggestion
        peopleCollectionView.register(cellType: PersonCell.self)
```